.freebsd.test.py

```diff
@@ -31,13 +31,18 @@
 
 def main():
     judgeenv.env['runtime'] = {}
-    judgeenv.env['extra_fs'] = {'PERL': ['/dev/dtrace/helper$'], 'RUBY2': ['/dev/dtrace/helper$']}
+    judgeenv.env['extra_fs'] = {
+        'PERL': [{"exact_file": "/dev/dtrace/helper"}],
+        'RUBY2': [{"exact_file": "/dev/dtrace/helper"}]
```

Comment:
Nit: end this with a `,`, such that the next time a commit has to add a language here, only the new line will appear in the diff.